### PR TITLE
Fix SDL checks for `main`

### LINKED ISSUE #14201: Signed builds are failing due to SDL checks in `main`
Signed builds are failing due to 'nuget.org' being among the package sources in one of the test projects

### PR DESCRIPTION
**Pull request by vzarytovskii**
Fixes #14201

### CODE REVIEW

File: tests/projects/SelfContained_Trimming_Test/check.ps1 [hunk 1]
```diff
@@ -19,5 +19,5 @@ $file = Get-Item .\bin\Release\net7.0\win-x64\publish\FSharp.Core.dll
 $file_len = $file.Length
 if (-not ($file_len -eq $expected_len))
 {
-    Write-Error "Test failed with unexpected FSharp.Core length:`nExpected:`n`t${expected_len} Bytes`nActual:`n`t${file_len} Bytes" -ErrorAction Stop
+    Write-Error "Test failed with unexpected FSharp.Core length:`nExpected:`n`t${expected_len} Bytes`nActual:`n`t${file_len} Bytes`nEiter codegen or trimming logic have changed. Please investigate and either update expected size or report an issue." -ErrorAction Stop
```
Comment (abonie) [inline tests/projects/SelfContained_Trimming_Test/check.ps1:22]: typo: nEiter

Comment (abonie) [reply tests/projects/SelfContained_Trimming_Test/check.ps1:22]: Also I believe "neither" should go with "nor" (rather than "or")

Comment (vzarytovskii) [inline tests/projects/SelfContained_Trimming_Test/check.ps1:22]:
```suggestion
    Write-Error "Test failed with unexpected FSharp.Core length:`nExpected:`n`t${expected_len} Bytes`nActual:`n`t${file_len} Bytes`nEither codegen or trimming logic have changed. Please investigate and update expected dll size or report an issue." -ErrorAction Stop
```

**Review by abonie — APPROVED** [review]

**Review by psfinaki — APPROVED** [review]

File: tests/projects/SelfContained_Trimming_Test/check.ps1 (hunk 1 above)
Comment (vzarytovskii) [reply tests/projects/SelfContained_Trimming_Test/check.ps1]: It's not neither. It's a line break `n and a word Either.

**Review by T-Gro — APPROVED** [review]

**Pull request merged**